logging/tests/unit/handlers/test_app_engine.py

```diff
@@ -30,15 +30,18 @@ def _make_one(self, *args, **kw):
         return self._get_target_class()(*args, **kw)
 
     def test_constructor(self):
-        from google.cloud.logging.handlers.app_engine import _GAE_PROJECT_ENV
+        from google.cloud.logging.handlers.app_engine import (
+            _GAE_PROJECT_ENV_STANDARD)
         from google.cloud.logging.handlers.app_engine import _GAE_SERVICE_ENV
         from google.cloud.logging.handlers.app_engine import _GAE_VERSION_ENV
 
         client = mock.Mock(project=self.PROJECT, spec=['project'])
 
-        with mock.patch('os.environ', new={_GAE_PROJECT_ENV: 'test_project',
-                                           _GAE_SERVICE_ENV: 'test_service',
-                                           _GAE_VERSION_ENV: 'test_version'}):
+        with mock.patch('os.environ', new={
+            _GAE_PROJECT_ENV_STANDARD: 'test_project',
+            _GAE_SERVICE_ENV: 'test_service',
+            _GAE_VERSION_ENV: 'test_version',
+        }):
```

Comment:
We really should test for the case that the flex variable is set, and that it overrides the standard one.

Reply:
Done in #6138.